### PR TITLE
add palindrome_number and test

### PR DESCRIPTION
**Pull request by lin6692**
Failed two test I don't know why. Please help.

### CODE REVIEW

File: functions/palindrome_number.py [hunk 1]
```diff
@@ -1,2 +1,12 @@
 def palindrome_number(x):
-    pass
+    temp = x
+    reverse = 0
```
Comment (Liqingcathy) [inline functions/palindrome_number.py:3]: we can initialize variables in one line. 
temp, reverse = x, 0

Comment (lin6692) [reply functions/palindrome_number.py:3]: No, I think my code is perfect.

Comment (Liqingcathy) [reply functions/palindrome_number.py:3]: Fine, whatever.

**Comment by Liqingcathy** [comment]
added original function at the end.

**Pull request merged**